config/consul_registr/registr.json

```diff
@@ -0,0 +1,26 @@
+{
+    "services": [
+        {
+        "name": "db",
+        "tags": ["postgres"],
+        "port": 5432,
+        "checks": [
+            {
+                "name": "Postgres-tcp-5432",
+                "tcp": "db:5432",
+                "interval": "10s",
+                "timeout": "1s"
+              }]
+        },{
+        "name": "graylog",
+        "tags": ["graylog"],
+        "port": 9000,
+        "checks": [
+            {
+                "name": "graylog-check-9000",
+                "tcp": "graylog:9000",
+                "interval": "30s",
+                "timeout": "5s"
+            }]
+        }]
+    }
```

Comment:
Move this to infra/consul
It is not a part of positioning-service config

Reply:
Done